cmake/onnxruntime_webassembly.cmake

```diff
@@ -11,18 +11,19 @@ add_executable(onnxruntime_wasm
   ${onnxruntime_wasm_src}
 )
 
-add_compile_definitions(
-  MLAS_NO_ONNXRUNTIME_THREADPOOL
-)
+if (NOT onnxruntime_ENABLE_WEBASSEMBLY_THREADS)
+  add_compile_definitions(
+    MLAS_NO_ONNXRUNTIME_THREADPOOL
+  )
 
-string(APPEND CMAKE_CXX_FLAGS_DEBUG " -g4 --source-map-base http://localhost:8000/")
+  # Override re2 compiler options to remove -pthread
+  set_property(TARGET re2 PROPERTY COMPILE_OPTIONS )
```

Comment:
is there possible other compile options for re2 accidentally removed?

Reply:
re2 cmake adds only -std=c++11 and -pthread as compile options. These two options are set by onnxruntime cmake.